operate/client/src/App/Processes/MigrationView/BottomPanel/index.tsx

```diff
@@ -96,7 +133,9 @@ const BottomPanel: React.FC = observer(() => {
                     <SourceFlowNodeName>
                       {sourceFlowNode.name}
                     </SourceFlowNodeName>
-                    {!isMapped && <Tag type="blue">Not mapped</Tag>}
+                    {isTargetSelected && !isMapped && (
+                      <Tag type="blue">Not mapped</Tag>
+                    )}
```

Comment:
I added `isTargetSelected` here to show the not mapped tags only if a target process version is selected. This caused and change in the UI which made updating the visual regression snapshots necessary